vignettes/r2dii-analysis.Rmd

```diff
@@ -12,117 +12,121 @@ knitr::opts_chunk$set(
 
 ## Load your r2dii libraries
 
-The first step in your analysis will be to load in the recommended r2dii packages into your current R session. [r2dii.data](https://2degreesinvesting.github.io/r2dii.data) includes fake data to help demonstrate the tool and [r2dii.match](https://2degreesinvesting.github.io/r2dii.match) provides functions 
-to help you easily match your loanbook to asset-level data.
+The first step in your analysis will be to load in the recommended r2dii packages into your current R session. [r2dii.data](https://2degreesinvesting.github.io/r2dii.data) includes fake data to help demonstrate the tool and [r2dii.match](https://2degreesinvesting.github.io/r2dii.match) provides functions to help you easily match your loanbook to asset-level data.
 
-``` {r r2dii libraries}
+``` {r use-r2dii}
```

Comment:
I think the minus symbol "-" is the one Rmarkdown prefers. Underscore "_" doesn't work well, and I'm afraid of useing space " ".